sdk/src/core/Cache.ts

```diff
@@ -22,27 +22,30 @@ export default class Cache {
         return this;
     }
 
-    public async setItem(key, data) {
-        this.setItemSync(key, data);
+    public getItemSync(key) {
+        const item = this._externals.localStorage.getItem(this._prefixKey(key));
```

Comment:
What if storage does not support sync?

Reply:
I think this is api is only for backward compatibility. We should avoid to use sync api

Reply:
https://github.com/ringcentral/ringcentral-js/blob/c7ba3e61ee3052ee42c8b0a5e3ff06862542d4d1/subscriptions/src/subscription/CachedSubscription.ts#L15 this method is still used here.

You have to rewrite the `Subscription` class to make it async. Otherwise I cannot accept this PR.

This would be a major version update, btw, since changes are not backward compatible.

Reply:
Yes, That's what I did in my earliest PR commit, which would have been a breaking change. 

@embbnux What do you think about it?

Reply:
@kirill-konshin Updated.